dockstore-cli-integration-testing/pom.xml

```diff
@@ -56,6 +56,14 @@
             <groupId>io.dockstore</groupId>
             <artifactId>dockstore-webservice</artifactId>
             <version>${dockstore-core.version}</version>
+            <exclusions>
+                <exclusion>
+                    <!-- Solves the failure caused by two liquibase.build.properties files
+                     being included in the final JAR   -->
```

Comment:
The file is in two separate JARs, not in the same JAR twice. And "the final JAR" may not be clear.

Suggest something like: "Liquibase is already bundled in the dockstore-webservice JAR".

Reply:
Updated comment